docs/extensions/basics/web-assets.md

```diff
@@ -110,8 +110,8 @@ In the above example:
   * `setFileName()` sets the file name
   * `setLate(true)` tells the asset injector to insert the `<link rel="stylesheet" href="path/web/koala.css">`
     at the end of the HTML `<body>`
-  * `setPriority(99)` tells the injector to insert this `<link rel="stylesheet" href="path/web/koala.css">`
-    *after* any other stylesheet tags in that location with a priority lower than 99
+  * `setPriority(5)` tells the injector when to insert the stylesheet. Lower
```

Comment:
Oh, I see … it looks like we either have a :bug: in core, or a :koala: :bug: 

Basically, if you have a second/set up to double check on, as with Silex/Symfony event listeners, the higher the number, the sooner it should load/apply, i.e. 1 is a higher priority that -1 

BTW, the change seems good otherwise, and clearer, lets just figure out what is going on here and merge after that.